huggle/Localization/en.xml

```diff
@@ -790,6 +790,7 @@
   <string name="query-protected-link">Protected link</string>
   <string name="queue-empty-title">It's empty</string>
   <string name="queue-empty-text">There is nothing else in the queue...</string>
+  <string name="protip">ProTip</string>
```

Comment:
Mmm IMO _ProTip_ can be translated in French as _Astuce_. For me a translation is needed. As you want.